types/net_props.d.ts

```diff
@@ -1,6 +1,5 @@
 import { NetFields } from './net_fields';
 
-
 type Params<T extends string> =
```

Comment:
Technically this is my change; I came through looking for `^import.*\n\n` since there were a couple places where the auto fix didn't do a great job at cleaning the mess it made.